package.json

```diff
@@ -61,6 +61,7 @@
     "primeflex": "^3.1.3",
     "primeicons": "^5.0.0",
     "primevue": "^3.32.1",
+    "sharp": "^0.33.3",
```

Comment:
Why did we add this?

Reply:
It gave an error for running that I needed to install that, but I can definitely take it out